index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" data-theme="lemonade">
+<head>
+    <link href="https://cdn.jsdelivr.net/npm/daisyui@4.11.1/dist/full.min.css" rel="stylesheet" type="text/css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+    <h1 class="hero text-3xl font-bold bg-base-200">
+        <div class="hero-content flex-col lg:flex-row" style="gap:0rem">
+            <img src="./icons/teapot.svg" class="size-14"/>Tea Time Game
+        </div>
+    </h1>
+    <div class="hero bg-base-200" style="min-height:60vh">
+        <div class="hero-content flex-col lg:flex-row-reverse">
+            <video width="320" height="230" controls class="rounded-lg shadow-2xl" style="width:50vw">
```

Comment:
This might be a quirk with how Firefox handles missing video files, but this is what the page looked like in Firefox PC. If it's a missing video quirk, probably a non-issue for us

![image](https://github.com/danipurdes/playteatime/assets/3577548/f5e25dc3-f6fd-4cee-984b-b2a5884a2260)

Reply:
I added a placeholder video to test and it looks ok now. Can you confirm?

Reply:
looks great, and the video choice is amazing too 🐄